contracts/collection.scilla

```diff
@@ -225,7 +228,7 @@ end
 
 procedure RequireValidMaxBPS(val_bps: Uint128)
   (* max: 25% (arbitrary number) *)
```

Comment:
Please also update the comment here to reflect the change as well as the contract documentation: 
https://github.com/Zilliqa/marketplace-contracts/blob/RIAL-787_update_commission_fee_range/contracts/README.md#11-setmaxcommissionfeebps

Reply:
Nice! but you missed updating the value inside the parenthesis (should be 50%), pls update as well:
"An admin function that lets `contract_owner` update the `max_commission_fee_bps` value. It can not exceed a hardcoded value set to 5000 (25%)."